src/components/gltf-model-plus.js

```diff
@@ -11,6 +11,8 @@ import { RGBELoader } from "three/examples/jsm/loaders/RGBELoader";
 import { BasisTextureLoader } from "three/examples/jsm/loaders/BasisTextureLoader";
 import { DRACOLoader } from "three/examples/jsm/loaders/DRACOLoader";
 import { GLTFLoader } from "three/examples/jsm/loaders/GLTFLoader";
+import GLBRangeRequests from "three-gltf-extensions/loaders/GLB_range_requests/GLB_range_requests";
```

Comment:
Note for reviews:

This is a `GLTFLoader` plugin for HTTP range requests. It does

1. checks if a file a url referes to is GLB and the server supports HTTP range requests
2. if so it first loads only json chunk and partially and lazily loads bin chunk content with HTTP range requests

https://github.com/takahirox/three-gltf-extensions/blob/main/loaders/GLB_range_requests/GLB_range_requests.js